04.ls/lib/ls_command.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require_relative 'entry'
+require_relative 'formatter'
+
+class LsCommand
+  def initialize(path)
+    path ||= '.'
+    @path = File.expand_path(path)
```

Comment:
```suggestion
    @path = File.expand_path(path || '.')
```

引数に再代入しない方がいいかも
https://bootcamp.fjord.jp/pages/avoid-re-assign